src/components/search/concept-modal/tabs/explanation.js

```diff
@@ -172,10 +172,10 @@ export const ExplanationTab = ({ result }) => {
                         fontSize: 15,
                         fontWeight: 500
                     }}>
-                        Score breakdown
+                        Explanation for this concept&apos;s relation to search term
                     </Text>
                     <div>
-                        <Text style={{ fontSize: 13 }} italic>Why am I seeing this result?</Text>
+                        <Text style={{ fontSize: 13 }} italic>What does the total score returned by the search mean??</Text>
```

Comment:
Are two ? intended?

Reply:
@hina-shah this was pushed through, did you mean to do this??

Reply:
Oh no, I'm sorry no. But I see that you fixed it. Thanks!